Resources/Prototypes/_SpaceCats/Entities/Clothing/Uniforms/jumpsuits.yml

```
@@ -0,0 +1,53 @@
+  parent: ClothingUniformBase
+  id: ClothingUniformJumpsuitBridgeOfficer
+  name: bridge officer uniform
+  description: More stylish than advanced, this formal suit serves as a great display of power.
+  components:
+  - type: Sprite
+    sprite: _SpaceCats/Clothing/Uniforms/Jumpsuit/bridgeofficer.rsi
+  - type: Clothing
+    sprite: _SpaceCats/Clothing/Uniforms/Jumpsuit/bridgeofficer.rsi
+  - type: Tag
+    tags:
+    - WhitelistChameleon
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)